crux-lucene/checkouts/crux-core

```diff
@@ -0,0 +1 @@
+../../crux-core
```

Comment:
do we need these checkouts? implies you're running your REPL from `crux-lucene` rather than the project root?

Reply:
I do yeah :-) is that not OK? Repl runs faster, dog-fooding in that module.